### DISCUSSION TITLE
Sometime, I have 5 seconds latency before Neo-Launcher is displayed, how can I fix it?

### DISCUSSION THREAD

**Discussion by stephane-klein** [otakuhqz/Omega#329] [Q&A] [answered]
Hello,

I use Neo-Launcher on my Oppo Find X3 Lite 5G, with ColorOS 13 and I have a Neo Launcher start latency.

Here is how I can reproduce this issue: I go to Firefox, I use it several minutes, and when I come back to Neo-Launcher I need to wait around 5 seconds before Neo-Launcher is displayed 🙁.

How can I fix it?

Best regards,
Stéphane

Same question on Reddit https://old.reddit.com/r/androidapps/comments/10udpgr/sometime_i_have_5_seconds_latency_before/

**Comment by machiav3lli** [comment]
Some OEMs seems to have optimized their Android forks causing apps being killed. Neither Saul or I have such devices, and so we can't really detect and solve such bugs on our own. And its clear that some OEMs do this intended to lock the users in their launchers collecting data.

**Comment by xrad** [comment]
You can check https://dontkillmyapp.com/oppo for ways to keep the Neo from being killed. Note though that this really is about battery optimization. If you can, check the adb logcat to ensure this is not in fact Neo launcher crashing/reloading, in which case you might post the crash log here.

**Comment by stephane-klein** [comment, marked as answer]
@mariachini @xrad fixed: in Android app configuration, I have enabled "Allow application to work in background".
